figures/compat.py

```diff
@@ -1,7 +1,9 @@
 """Figures compatability module
 
-This module serves to provide a common access point to functionality that
-differs from  different named Open edX releases
+This module serves to provide a common access point to edx-platform objects.
```

Comment:
**Nit:** The module can now be called `openedx` instead of `compat` to reflect the newer name imho.

Reply:
Good idea, I'll do that during another refactor